src/app/app.module.ts

```diff
@@ -63,6 +63,7 @@ import { AssignmentFormComponent } from './views/assignments/assignment-form/ass
 import { ViewAssignmentComponent } from './views/assignments/view-assignment/view-assignment.component';
 import { CountdownModule } from 'ngx-countdown';
 import { AddStudentsComponent } from './views/assignments/add-students/add-students.component';
+import { CustomDialogConfirmationComponent } from './shared/components/custom-dialog-confirmation/custom-dialog-confirmation.component';
```

Comment:
Is this the dialog for Preview test ? if yes put it in the assignment/popup folder instead of shared

Reply:
No its is common component we can levarage it in future

Reply:
This reminds - will this same popup come up when student clicks on the take test on his dashboard?